bob-api/api/de_onderwijzers_api.php

```diff
@@ -0,0 +1,195 @@
+<?php namespace api;
+
+use ErrorException;
+
+/**
+ * Class DeOnderwijzers
+ *
+ * Wrapper for de API die de onderwijzers beschikbaar hebben gesteld.
+ *
+ * Documentatie die goed is om te lezen:
+ * - https://graphql.org/learn/queries
```

Comment:
volgens mij is het beter om bij de linkjes te vertellen wat het doet.

Reply:
Mee eens, doe ik!